src/lib/capability.test.ts

```diff
@@ -100,11 +100,11 @@ describe("Capability", () => {
     const capability = new Capability(capabilityConfig);
 
     const storeResult = capability.registerStore();
-    expect(storeResult).toHaveProperty("store");
+    expect(storeResult).toBeDefined();
     expect(mockLog.info).toHaveBeenCalledWith(`Registering store for ${capabilityConfig.name}`);
 
     const scheduleStoreResult = capability.registerScheduleStore();
-    expect(scheduleStoreResult).toHaveProperty("scheduleStore");
+    expect(scheduleStoreResult.onReady).toBeDefined;
```

Comment:
```suggestion
    expect(scheduleStoreResult.onReady).toBeDefined();
```
Missing the invocation.

Reply:
Fixed